linter.py

```diff
@@ -71,24 +111,50 @@ def split_match(self, match):
 
         if match:
             open_file_name = os.path.basename(self.view.file_name())
-            linted_file_name = match.group('file_name')
+            # Since the filename on the top row might be different than the open file if, for example,
+            # something is imported from another file. Use the filename from the footer is it's available.
+            linted_file_name = match.group('file_name_2') or match.group('file_name_1')
 
             if linted_file_name == open_file_name:
+
                 message_title = match.group('message_title')
                 message = match.group('message')
-                message_footer = match.group('message_footer') or ""
-
-                if message_title and message_title.strip():
-                    message = '"{0}"" {1} {2}'.format(
-                        message_title,
-                        message,
-                        message_footer
-                    )
-
-                line = max(int(match.group('line')) - 1, 0)
-                col = int(match.group('col')) - 1
+                message_footer = match.group('message_footer')
+                col = match.group('col_2') or match.group('col_1')
+                message_format = []
+
+                if message_title:
+                    message_format.append('[ {0} ]')
+                if message:
+                    message_format.append('{1}')
+                if message_footer:
+                    message_format.append('[ {2} ]')
+
+                message = " ".join(message_format).format(
+                    message_title,
+                    message,
+                    message_footer
+                )
+
+                # Get the start and ending indexes of the line and column
+                line_cols = col.replace(':', ',').split(',')
```

Comment:
Unless there is some technical limitation I'm totally missing that doesn't make sense to me. I see no reason named capture groups vs splitting would lead to different formatting of the error message.